Gulpfile.js

```diff
@@ -0,0 +1,23 @@
+var babel  = require('gulp-babel');
+var eslint = require('gulp-eslint');
+var gulp   = require('gulp');
+var rimraf = require('rimraf');
+
+
+gulp.task('clean', function (cb) {
+    rimraf('lib', cb);

```

Comment:
Use the [`del` module](https://www.npmjs.com/package/del) instead of `rimraf`. You can get rid of callback, because `del` returns a `promise`
```js
var del = require('del');

gulp.task('del', function () {
    return rimraf('lib');
});
```